packages/docs/src/components/edit-link.js

```diff
@@ -32,7 +31,7 @@ export const EditLink = ({ base, children, ...props }) => (
 
 EditLink.defaultProps = {
   base:
-    'https://github.com/system-ui/theme-ui/edit/master/packages/docs/src/pages',
```

Comment:
Thanks, but this looks like it'd "fix" the link for pages under `/recipes` but break other pages. I think the edit link should either work for both cases or just be disabled for pages in `/recipes`

Reply:
Thanks a lot for your review sorry for that 🙈 I updated the pull request. It's not a pretty solution so far but I think it works for now and I like the idea of quickly creating pull-request with such a link :)